askbot/static/askbot/css/user_profile.scss

```diff
@@ -0,0 +1,83 @@
+&.user-profile-page {
+    #ContentLeft {
+        .section-title {
+            float: left;
+        }
+
+        .tabBar-profile {
+            width: auto;
+            margin-top: 21px;
+            margin-bottom: 10.5px;
+            margin-left: 35px;
+        }
+
+        .user-info-table {
+            .avatar {
+                margin: 0 15px 0 0;
+
+                .avatar-box .gravatar {
+                    border: none;
+                    border-radius: 0;
```

Comment:
I talked with CJ and she said that she made a mistake and the profile image should have a radius.